gaia-rs/src/lib.rs

```diff
@@ -139,14 +138,14 @@ impl AuxHandler for GaiaCoreClient {
         match cmd {
             GaiaAuxCmd::Genutil(cmd) => match cmd {
                 genutil::cmd::GenesisCmd::CollectGentxs(cmd) => {
-                    let (_, genesis) = genutil::collect_txs::gen_app_state_from_config(
-                        cmd,
-                        &GaiaStoreKey::Bank,
-                        "genutil",
-                    )?;
+                    let (_, genesis) =
```

Comment:
The `gentx` and `collect-gentx` are added under the `genutil` sub-command rather than a sub-command of `gaia-rs`

Reply:
flatten cli